profiler/src/main/java/com/navercorp/pinpoint/profiler/monitor/codahale/gc/CmsCollector.java

```diff
@@ -60,20 +68,26 @@ public CmsCollector(MetricMonitorRegistry registry) {
         this.heapNonHeapMax = getLongGauge(gauges, JVM_MEMORY_NONHEAP_MAX);
         this.heapNonHeapUsed = getLongGauge(gauges, JVM_MEMORY_NONHEAP_USED);
 
+        this.codeCacheUsage = getLongGauge(gauges, JVM_MEMORY_POOLS_CMS_CODE_CACHE);
+        this.edenSpaceUsage = getLongGauge(gauges, JVM_MEMORY_POOLS_CMS_EDEN);
+        this.oldGenUsage = getLongGauge(gauges, JVM_MEMORY_POOLS_CMS_OLDGEN);
+        this.survivorSpaceUsage = getLongGauge(gauges, JVM_MEMORY_POOLS_CMS_SURVIVOR);
+        this.permGenUsage = getLongGauge(gauges, JVM_MEMORY_POOLS_CMS_PERMGEN);
+        this.metaspaceUsage = getLongGauge(gauges, JVM_MEMORY_POOLS_CMS_METASPACE);
+
         this.gcCount = getLongGauge(gauges, JVM_GC_CMS_COUNT);
         this.gcTime = getLongGauge(gauges, JVM_GC_CMS_TIME);
-
     }
 
     @Override
     public int getTypeCode() {
-        return GC_TYPE.getValue();
+        return GC_TYPE.ordinal();
```

Comment:
Using `getValue()` is definitely the better option. Not sure why they were using `ordinal()`.

They both should be returning the same value so I don't think there would be any issues with backwards compatibility.